src/ui/buttons.jsx

```diff
@@ -85,18 +90,23 @@ class Buttons extends React.PureComponent {
         >
           {colorPickerText}
         </button>
+        <button onClick={toggleChartList} type="button">
+          {chartListText}
+        </button>
```

Comment:
Could you move this button to be immediately after "View Entrances" and `<br />` right after the "View Charts" button?